documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -259,6 +259,20 @@
               description: 'Prometheus %(prometheusName)s has missed {{ printf "%%.0f" $value }} rule group evaluations in the last 5m.' % $._config,
             },
           },
+          {
+            alert: 'PrometheusTargetLimitHit',
+            expr: |||
+              prometheus_target_scrape_pool_targets_added{%(prometheusSelector)s} > prometheus_target_scrape_pool_targets_added{%(prometheusSelector)s} > 0
```

Comment:
You still have added here.

Reply:
More generally, it'd be safer to work from the counter.

Reply:
The counter would resolve itself; which means the alert would not be firing all the time.

Reply:
I'm not sure what you mean by that. Gauges will miss things that happen between scrapes, and this expression is not resilient to failed scrapes. Counters avoid both of those problems.